Crash in `ischannel_func`

Steps to reproduce:

1. Start irssi
2. Connect to Freenode: `/connect -ssl chat.freenode.net 6697`. Crash.

I'm using OS X with clang.

Backtrace: https://gist.github.com/ahf/3cc930a340ec070a5041

I suspect this is related to your recent patches @LemonBoy - up for looking at it?

Can't reproduce in linux.

The bad access claims to happen in 0x50fd50, frame #1 has the target parameter in 0x000000010050fd50.

Took me a while to find it - switched to clang and enabled all hardening flags hoping that ASLR kicks in and i get the heap in a higher memory address... and it still doesn't crash for me, even with the heap at 0x555555852000.

But I did get these warnings from clang!

```
fe-irc-messages.c:49:11: warning: implicit declaration of function 'fe_channel_skip_prefix' is invalid in C99 [-Wimplicit-function-declaration]
        target = fe_channel_skip_prefix(IRC_SERVER(server), target);
                 ^
fe-irc-messages.c:49:9: warning: incompatible integer to pointer conversion assigning to 'const char *' from 'int' [-Wint-conversion]
        target = fe_channel_skip_prefix(IRC_SERVER(server), target);
               ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
fe-irc-messages.c:112:9: warning: incompatible integer to pointer conversion assigning to 'const char *' from 'int' [-Wint-conversion]
        target = fe_channel_skip_prefix(IRC_SERVER(server), target);
               ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
fe-irc-messages.c:140:9: warning: incompatible integer to pointer conversion assigning to 'const char *' from 'int' [-Wint-conversion]
        target = fe_channel_skip_prefix(IRC_SERVER(server), target);
               ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

And that certainly sounds like what I suspected (ints are 32 bit, pointers are 64 bit, and some type downgrading happened). Still no idea why it works for me.

Adding the missing `#include` should fix it.

--- a/src/fe-common/irc/fe-events.c
+++ b/src/fe-common/irc/fe-events.c
@@ -41,6 +41,7 @@
 #include "fe-queries.h"
 #include "fe-windows.h"
 #include "fe-irc-server.h"
+#include "fe-irc-channels.h"
 
 static void event_privmsg(IRC_SERVER_REC *server, const char *data,
 			  const char *nick, const char *addr)

--- a/src/fe-common/irc/fe-irc-messages.c
+++ b/src/fe-common/irc/fe-irc-messages.c
@@ -36,6 +36,7 @@
 
 #include "fe-queries.h"
 #include "window-items.h"
+#include "fe-irc-channels.h"
 
 static void sig_message_own_public(SERVER_REC *server, const char *msg,
 				   const char *target, const char *origtarget)

